crates/subspace-farmer/src/plot.rs

```diff
@@ -644,10 +645,10 @@ impl IndexHashToOffsetDB {
             .map(PieceDistance::from_big_endian)
             .unwrap_or(PieceDistance::MIDDLE);
 
-        Ok(Range {
-            start: self.piece_distance_to_hash(start),
-            end: self.piece_distance_to_hash(end),
-        })
+        Ok(Some(RangeInclusive::new(
```

Comment:
Above code still uses `.unwrap_or()`, which was the reason to have an option here.